### PR TITLE
Fix Upgrade Gardener Pipelines

### PR DESCRIPTION
**Pull request by jeremyharisch**
**Description**

The working directory needs to be differenet than the extra_Ref directory - Because when using `--source` flag with anything different than `local`, it cleans up the directory afterwards. 
In those two pipeline we need multiple versions of Kyma, due to the upgrade scenario, plus we need the fast-integration tests. Thus, we will use source flag for the Kyma version and the external_refs for the fast-integration test.

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Comment by jeremyharisch** [comment]
\hold

File: vpath/pjtester.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+pjNames:
+  - pjName: "kyma-upgrade-gardener-kyma-to-kyma2"
+  - pjName: "kyma-upgrade-gardener-kyma2-to-main"
+prConfigs:
+  kyma-project:
+    test-infra:
+      prNumber:
```
Comment (Halamix2) [inline vpath/pjtester.yaml]:
```suggestion
```

Comment (jeremyharisch) [reply vpath/pjtester.yaml]: oh thank you

**Review by tobiscr — APPROVED** [review]

**Pull request merged**